client/src/components/related-items/RelateAndCompare.js

```diff
@@ -12,14 +13,23 @@ class RelateAndCompare extends React.Component {
   }
 
   render() {
+    const { relatedProducts } = this.props;
     return (
       <div data-testid="relate-compare">
-        <RelatedProducts relatedProducts={data.related} />
-        <YourOutfit savedOutfit={data.related} />
+        <RelatedProducts relatedProducts={relatedProducts} />
+        <YourOutfit savedOutfit={relatedProducts} />
         {/* <ComparisonModal /> */}
       </div>
     );
   }
 }
 
-export default RelateAndCompare;
+const mapStateToProps = (state) => {
+  return {
+    relatedProducts: state.relatedProducts
+  };
+};
+
+export default connect(mapStateToProps, {
+  setRelatedProduct
```

Comment:
Does this work? I don't see `dispatch` being mapped anywhere?

Reply:
I think connect uses dispatch under the hood to dispatch actions.

Reply:
cool, didn't know you could use the API like this!